test/selenium/pages/MonthlyContribution.scala

```diff
@@ -0,0 +1,55 @@
+package selenium.pages
+
+import org.scalatest.selenium.Page
+import selenium.util.{Browser, Config}
+
+object MonthlyContribution extends Page with Browser {
+
+  val url = s"${Config.supportFrontendUrl}/monthly-contributions"
+
+  private val stripeButton = id("qa-pay-with-card")
```

Comment:
As discussed in person, I think using dedicated ids to make the code more testable is worth the additional code in this case (I believe it's considered best practice for this type of testing). Although I think your suggested approach is possible, I think it will make the tests too brittle...